support/map.c

```diff
@@ -4716,52 +4716,36 @@ int Mono_Posix_ToPosixMadviseAdvice (int x, int *r)
 int Mono_Posix_FromSeekFlags (short x, short *r)
 {
 	*r = 0;
-	if (x == Mono_Posix_SeekFlags_L_INCR)
 #ifdef L_INCR
+	if (x == Mono_Posix_SeekFlags_L_INCR)
```

Comment:
`support/map.c` is generated by [`create-native-ma`](https://github.com/mono/mono-tools/tree/master/create-native-map), and these changes don't look like `create-native-map` output.

As such, they will be lost during the next time `make -C mono/support refresh` is executed.